src/Ilios/CoreBundle/Tests/Controller/AamcMethodControllerTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Ilios\CoreBundle\Tests\Controller;
+
+use FOS\RestBundle\Util\Codes;
+
+/**
+ * AamcMethod controller Test.
+ * @package Ilios\CoreBundle\Test\Controller;
+ */
+class AamcMethodControllerTest extends AbstractControllerTest
+{
+    /**
+     * @return array
+     */
+    protected function getFixtures()
+    {
+        return [
+            'Ilios\CoreBundle\Tests\Fixture\LoadAamcMethodData',
+            'Ilios\CoreBundle\Tests\Fixture\LoadSessionTypeData'
+        ];
+    }
+
+    public function testGetAamcMethod()
+    {
+        $aamcMethod = $this->container
+            ->get('ilioscore.dataloader.aamcmethod')
+            ->getOne()['aamcMethod']
```

Comment:
I think ::getOne() should return an array of the data - not an array keyed by 'aamcMethod' which we have to extract.